internal/commands/image/rm.go

```diff
@@ -0,0 +1,58 @@
+package image
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/docker/app/internal/store"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/distribution/reference"
+	"github.com/spf13/cobra"
+)
+
+func rmCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "rm [APP_IMAGE] [APP_IMAGE...]",
+		Short: "Remove an application image",
+		Args:  cli.RequiresMinArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			appstore, err := store.NewApplicationStore(config.Dir())
+			if err != nil {
+				return err
+			}
+
+			bundleStore, err := appstore.BundleStore()
+			if err != nil {
+				return err
+			}
+
+			errs := []string{}
+			for _, arg := range args {
+				if err := runRm(bundleStore, arg); err != nil {
+					errs = append(errs, fmt.Sprintf("Error: %s", err))
+				}
+			}
+			if len(errs) > 0 {
+				return errors.New(strings.Join(errs, "\n"))
+			}
+			return nil
+		},
+	}
+}
+
+func runRm(bundleStore store.BundleStore, app string) error {
+	ref, err := reference.ParseNormalizedNamed(app)
+	if err != nil {
+		return err
+	}
+
+	err = bundleStore.Remove(ref)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Deleted: " + ref.String())
```

Comment:
Perhaps we should consistently use `dockerCli.Out()` / `dockerCli.Err()` for these; _ideally_ we would print the reference on stdout, and `Deleted:` on stderr, but I don't think we do that currently on the CLI